test/TestExpectations.json

```diff
@@ -527,6 +541,20 @@
     "expectations": ["SKIP"],
     "comment": "Not implemented for BiDi yet."
   },
+  {
+    "testIdPattern": "[network.spec] network Response.fromCache should work for *",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["cdp", "firefox"],
+    "expectations": ["FAIL"],
+    "comment": "TODO: add a comment explaining why this expectation is required (include links to issues)"
+  },
+  {
+    "testIdPattern": "[network.spec] network Response.fromCache should work for *",
+    "platforms": ["darwin", "linux", "win32"],
+    "parameters": ["chrome", "webDriverBiDi"],
+    "expectations": ["FAIL", "PASS"],
```

Comment:
Note: This used to be a FAIL, PASS for a single test "[network.spec] network Response.fromCache should work". Now that we are running two similar tests, I used a wildcard which moved the expectation up. I haven't checked if the script version is actually FAIL/PASS on Chrome though.